### PR TITLE
Add a troubleshooting docs page

### PR DESCRIPTION
**Pull request by kwmonroe**
## Summary

We need a troubleshooting page for solutions to common problems. This PR creates that as a 'reference' and includes one such problem/solution. The objective of this PR is to get the target page in place; the team will fill in additional solutions as they arise.

## Additional scope

We also have a 'Support' how-to document, though it's pretty bare.  It felt like a place i'd go to find troubleshooting help, so I've updated that area.  If this is too far out-of-scope, i'm happy to remove.

### CODE REVIEW

File: docs/src/explanation/troubleshooting.md
Comment (bschimke95) [inline docs/src/explanation/troubleshooting.md:1]: Should we move that into `references`, similar to [microk8s](https://microk8s.io/docs/get-in-touch)?

**Review by bschimke95 — COMMENTED** [review]
Nice work @kwmonroe, added some comments. 
Up to @evilnick to approve on the structure.

File: docs/src/explanation/troubleshooting.md
Comment (louiseschmidtgen) [inline docs/src/explanation/troubleshooting.md:10]: Please add a link here to the upstream docs that explain what a kubeconfig file is.

File: docs/src/explanation/troubleshooting.md
Comment (louiseschmidtgen) [inline docs/src/explanation/troubleshooting.md:26]: Would this be the right spot to add example code on how to append the output of `k8s config` to an existing kubeconfig file?

File: docs/src/explanation/troubleshooting.md
Comment (louiseschmidtgen) [reply docs/src/explanation/troubleshooting.md:6]: +1 if we include the error it is more likely to get people directed to our docs when they search for it.

**Review by evilnick — COMMENTED** [review]
LGTM when the comments are addressed - I also think the headings should match the actual error message

File: docs/src/explanation/troubleshooting.md
Comment (kwmonroe) [reply docs/src/explanation/troubleshooting.md]: @louiseschmidtgen i don't think that fits well here -- at this point, you've already merged/appended the kubeconfig(s) and are dealing with the failure.

i think a separate page for tips and tricks / best practices would be a good place for managing external kubeconfigs, though that's out of scope for here.

**Comment by evilnick** [comment]
This is all going to go badly for conflicts on my PR, but thanks

**Review by evilnick — APPROVED** [review]
:cake:

**Pull request merged**
